torch/_functorch/_aot_autograd/jit_compile_runtime_wrappers.py

```diff
@@ -342,7 +363,11 @@ def aot_dispatch_autograd(
                 # Comparing ph_arg.stride() with real_stride directly may
                 # cause dynamic dimensions in ph_arg being specialized to static
                 # value. Using the hints to avoid that.
-                if _get_symint_hints(ph_arg.stride()) != real_stride:
+                if strides_meaningfully_different(
```

Comment:
the typical idiom is `significant strides` at least per `check_significant_strides`